tools-v2/pkg/cli/command/curvebs/stop/volumeSnapshot/stop_volume.go

```diff
@@ -0,0 +1,126 @@
+package volumeSnapshot
+
+import (
+	"encoding/json"
+	"strconv"
+	"time"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	cobrautil "github.com/opencurve/curve/tools-v2/internal/utils"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+)
+
+const (
+	VERSION = "0.0.6"
+	LIMIT   = "100"
+	OFFSET  = "0"
+)
+
+type Stop struct {
+	serverAddress []string
+	timeout       time.Duration
+
+	User     string
+	FileName string
+	UUID     string
+
+	messages []string
+}
+
+func newStopSnapShot(serverAddress []string, timeout time.Duration, user, fileName, uuid string) *Stop {
+	return &Stop{
+		serverAddress: serverAddress,
+		timeout:       timeout,
+		User:          user,
+		FileName:      fileName,
+		UUID:          uuid,
+	}
+}
+
+type Record struct {
+	UUID string `json:"UUID"`
+	User string `json:"User"`
+	File string `json:"File"`
+
+	Result string `json:"Result"`
+}
+
+func (s *Stop) queryStopBy() ([]*Record, *cmderror.CmdError) {
+	records, err := s.getSnapShotListAll()
+	return records, err
+}
+
+func (s *Stop) getSnapShotListAll() ([]*Record, *cmderror.CmdError) {
```

Comment:
The function is intend to list all snapshot? may be we can provide a command such as `curve bs list snapshot`, then we invoke the interface here.